src/app/zwiazek/kalendarz/_components/CalendarTable.tsx

```diff
@@ -0,0 +1,103 @@
+import clsx from "clsx";
+import { CalendarHeader } from "@/app/zwiazek/kalendarz/_components/CalendarHeader";
+import { formatDate } from "@/utils";
+import { cardsPlaceholder } from "@/app/zwiazek/kalendarz/_components/Events/List";
+import {
+  daysOfWeek,
+  today,
+} from "@/app/zwiazek/kalendarz/_components/Dates/Dates";
+
+type CalendarTableProps = {
+  currentDate: Date;
+  prevMonth: () => void;
+  nextMonth: () => void;
+};
+
+export const CalendarTable = ({
+  currentDate,
+  prevMonth,
+  nextMonth,
+}: CalendarTableProps) => {
+  const generateCalendar = (date: Date) => {
+    const year = date.getFullYear();
+    const month = date.getMonth();
+    const currentDate = new Date(year, month, 1);
+    const calendarDays = [];
+    const dayOfWeek = currentDate.getDay(); // Numer dnia tygodnia
+    const daysToMonday = dayOfWeek === 0 ? -6 : 1 - dayOfWeek;
+    currentDate.setDate(currentDate.getDate() + daysToMonday);
+
+    for (let i = 0; i < 6; i++) {
+      const week = [];
+      for (let j = 0; j < 7; j++) {
```

Comment:
Extract `6` and `7` to some const with meaningful name. Try to avoid `"magic numbers"`